dependency_management/set_lower_bounds

```diff
@@ -50,6 +50,7 @@ def _do_updates(explicit_dependencies, cargo_tree):
                     "-p",
                     "%s:%s" % (crate, current_version),
                     "--precise=%s" % version,
+                    "--manifest-path=%s" % manifest_path,
```

Comment:
Here too, ```manifest_path``` could be ```None```. Might as well document it in the parameter list for the method also, and make sure to check for ```None``` like you do elsewhere.

Reply:
Okay, I'm guessing this will need something like, `if manifest_path is None:`, but what should I do in that scenario?

I can't use `default=[]`, as with the `--ignore-missing` parameter in compare_fedora_versions, because that will result in a traceback, which fails with this:
```
error: the manifest-path must be a path to a Cargo.toml file
```

Should I just have the program gracefully display an error if manifest_path is None?

Reply:
...No, I shouldn't display an error.  Now I see what I did before, and a way to add the parameter if it's not None.  (I don't know if it's the best way, but it seems to work when I test it.)

Reply:
Resolved.